README.md

```diff
@@ -19,19 +19,24 @@ Fork this repo. After that, you have to clone your forked repo on your local mac
 
 Assuming you have already installed PostgreSQL, access to template1, wich is a default PostgreSQL database:
 ```
-sudo su postgres
-psql template1
+user@smachine:/smartCampuz$ sudo su postgres
+postgres@smachine:/smartCampuz$ psql template1
 ```
 
-Once you are loged in, you should create a user, a database and grant all permisions over the database to that user.
+Once you are loged in, you should create a user.
 ```
 template1=# CREATE USER <myUser> WITH PASSWORD '<myPassword>';
-template1=# CREATE DATABASE smartcampuz;
-template1=# GRANT ALL PRIVILEGES ON DATABASE smartcampuz to <myUser>;
 \q
 exit
 ```
 
+Without logging out from `posrtgres` account, execute the resetDB.sh script. This will create a database named 
```

Comment:
The expression used in **from \`posrtgres\` account** should be only to include a code fragment. If you want to highlight the "postgre" word  because is important to you , I recommend you to use **from \*postgres\* account**.